scripts/e2e/run-e2e.sh

```diff
@@ -26,22 +26,66 @@ function InstallAGIC() {
     helm repo add staging https://appgwingress.blob.core.windows.net/ingress-azure-helm-package-staging/
     helm repo update
 
-    # AAD pod identity is taking time to assign identity. Timeout is set to 60 sec
+    # delete agic if exist
+    helm delete agic-${version} -n agic || true
+
+    # AAD pod identity is taking time to assign identity. Timeout is set to 120 sec
+    helm upgrade --install agic-${version} staging/ingress-azure \
+    --set appgw.applicationGatewayID=${applicationGatewayId} \
+    --set armAuth.type=aadPodIdentity \
+    --set armAuth.identityResourceID=${identityResourceId} \
+    --set armAuth.identityClientID=${identityClientId} \
+    --set rbac.enabled=true \
+    --timeout 120s \
+    --wait \
+    -n agic \
+    --version ${version}
+
+    # create namespace for testing shared backend
+    kubectl create namespace test-brownfield-ns || true
+
+    # apply backends to test both non-prohibited and prohibited target
+    kubectl apply -f test-prohibit-backend.yaml
+
+}
+
+function SetupSharedBackend() {
+    
+    # delete agic with share disabled
+    helm delete agic-${version} -n agic || true
```

Comment:
do we need to delete ? Can you just update the current release ?

Reply:
removed